### ISSUE TITLE
Pond service: provide GET route which shows the status of the tanks

### ISSUE THREAD

**Issue by Terkwood** [Terkwood/prawnalith#45]
Pond service should provide all Redis data related to tanks, when queried at `GET /tanks`.

Temp & pH are fine to start with, though we may want to expose some of the other "trivia" data, later.

## Crib notes on data structures

Return value example
```javascript
[
     {
           "tank": { "id": 1, "name": "The Mothership" },
           "temp": { "f": 82.18, "c": 27.88 },
           "ph": { "val": 8.11, "mv": 500.15 }
     },
     {
           "tank": { "id": 2, "name": "The Pond" },
           "temp": { "f": 83.1, "c": 28.39 },
           "ph": { "val": 7.98, "mv": 488.33 }
     }
]
```

Rust struct example
```rust


/// A struct to hold some data from the HTTP request
/// for temp/ph info.
#[derive(Debug, Serialize, Deserialize)]
pub struct TankStatus {
   pub tank: Tank,
   pub temp: Temp,
   pub ph: Ph,
}

#[derive(Debug, Serialize, Deserialize)]
pub struct Tank {
   pub id: i32,
   pub name: String,
}

#[derive(Debug, Serialize, Deserialize)]
pub struct Temp {
   pub f: f32,
   pub c: f32,
}

#[derive(Debug, Serialize, Deserialize)]
pub struct Ph {
   pub val: f32,
   pub mv: f32,
}
```

**Comment by Terkwood** [comment]
Ended up using a flat data structure for the tanks, rather than the nested data structure described here. This approach is (A) flatter and (B) preserves consistent naming from Redis up to the frontend.

**Comment by Terkwood** [comment]
Implemented in #49

**Comment by Terkwood** [comment]
```
curl -k -H "Authorization: Bearer $FIREBASE_JWT" https://$FIREBASE_HOST/tanks | python -m json.tool
```  

```
[
    {
        "id": 1,
        "name": "The Mothership",
        "ph": 7.72,
        "ph_mv": 461.05,
        "ph_update_count": 2454127,
        "ph_update_time": 1542744006,
        "temp_c": 27.56,
        "temp_f": 81.61,
        "temp_update_count": 2535781,
        "temp_update_time": 1542744006
    },
    {
        "id": 2,
        "name": "The Pond",
        "temp_c": 24.62,
        "temp_f": 76.32,
        "temp_update_count": 191511,
        "temp_update_time": 1542744003
    }
]
```
